mix.exs

```diff
@@ -22,8 +30,18 @@ defmodule ElixirProbes.MixProject do
   defp deps do
     [
       {:plug, "~> 1.0"}
-      # {:dep_from_hexpm, "~> 0.3.0"},
-      # {:dep_from_git, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
+    ]
+  end
+
+  defp description do
+    "Common probes for Elixir applications in Kubernetes deployments"
+  end
+
+  defp package do
+    [
+      name: "pacer",
```

Comment:
😆  might want to change this one

Reply:
lol. Oops, I clearly did not mean to do that 😅